lib/msal-common/src/app/module/AuthorizationCodeModule.ts

```diff
@@ -402,14 +403,10 @@ export class AuthorizationCodeModule extends AuthModule {
      * @param codeResponse 
      */
     private async getTokenResponse(tokenEndpoint: string, tokenReqParams: ServerTokenRequestParameters, tokenRequest: TokenExchangeParameters, codeResponse?: CodeResponse): Promise<TokenResponse> {
-        // Perform token request.
-        const acquiredTokenResponse = await this.networkClient.sendPostRequestAsync<ServerAuthorizationTokenResponse>(
-            tokenEndpoint,
-            {
-                body: tokenReqParams.createRequestBody(),
-                headers: tokenReqParams.createRequestHeaders()
-            }
-        );
+        const clientInfo = buildClientInfo(this.cacheStorage.getItem(PersistentCacheKeys.CLIENT_INFO), this.cryptoObj);
```

Comment:
I am assuming `CLIENT_INFO` is already cached from the response here? Please throw if there is an error here, there should be no case of `CLIENT_INFO` being null as per eSTS instructions.

Reply:
Rather throw basically wherever you think you are reading `CLIENT_INFO` from the NW for the first response if not already done.

Reply:
what is NW in this statement?